pipes/Farbmosaik_pipe.py

```diff
@@ -0,0 +1,24 @@
+from pipes.pipe import Pipe
+import cv2
+
+
+class BGRtoGRB_pipe(Pipe):
+    def pipe(self, data):
+        orig = data.copy()
+        blur = data.copy()
+        height = orig.shape[0]
+        width = orig.shape[1]
+
+        cv2.blur(orig, (15,15), blur);
+        
+        data[:int(height/2),int(width/2):,1] = blur[int(height/2):,:int(width/2),0]
+        data[int(height/2):,:int(width/2),:2] = blur[:int(height/2),int(width/2):,:2]
+        data[int(height/2):,int(width/2):,2] = blur[:int(height/2),:int(width/2),1]
+        data[:int(height/2),:int(width/2),0] = blur[int(height/2):,int(width/2):,2]
```

Comment:
PEP8 shows, that you should put spaces around division symbols ("/") and behind commas! Anyways, really nice work!